packages/components/src/LightBox/LightBox.module.css

```diff
@@ -15,71 +15,143 @@
   position: absolute;
-  top: 50%;
-  z-index: var(--elevation-base);
-  transform: translateY(-50%);
+  top: 43%;
+  z-index: calc(var(--elevation-base) + 1);
 }
 
 .next {
-  right: var(--space-base);
+  right: var(--space-large);
+  @media (--medium-screens-and-up) {
+    right: var(--space-larger);
+  }
 }
 
 .prev {
-  left: var(--space-base);
+  left: var(--space-large);
+  @media (--medium-screens-and-up) {
+    left: var(--space-larger);
+  }
+}
+
+.buttonHidden {
+  opacity: 0;
+  transition: opacity var(--timing-base) ease-in;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+.buttonVisible {
+  opacity: 1;
+  transition: opacity var(--timing-base) ease-in;
 }
 
 .lightboxWrapper {
   display: flex;
   position: fixed;
   top: 0;
+  right: 0;
+  bottom: 0;
   left: 0;
   z-index: var(--elevation-modal);
-  width: 100%;
-  height: 100%;
-  justify-content: space-between;
+  max-width: 100%;
+  max-height: 100%;
+  padding: var(--space-base);
+  background-color: var(--color-base-grey--400);
+  justify-content: center;
   flex-direction: column;
+
+  @media (--medium-screens-and-up) {
+    padding: var(--space-base) var(--space-larger);
+  }
 }
 
 .toolbar {
-  display: flex;
+  display: grid;
   z-index: var(--elevation-base);
-  padding: var(--space-small) var(--space-base);
-  color: var(--color-text--secondary);
+  margin-bottom: var(--space-largest);
   font-size: var(--typography--fontSize-large);
-  background-color: var(--color-surface--background);
-  justify-content: space-between;
+  align-items: center;
+  grid-template-columns: 1fr 1fr 1fr;
 }
 
-.title {
-  display: inline-flex;
-  align-items: center;
+.toolbar button {
+  grid-column: 3;
+  justify-self: end;
 }
 
-.imagesWrapper {
-  display: flex;
-  justify-content: center;
+.toolbar p {
+  grid-column: 2;
+  justify-self: center;
+}
+
+.slideWrapper {
+  z-index: var(--elevation-base);
+  height: 100%;
+  max-width: 100%;
+  margin-bottom: var(--space-large);
+  @media (--medium-screens-and-up) {
+    padding: 0 calc(var(--space-largest) + var(--space-base)); /* nav button width + padding */
+  }
+}
+
+.image {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.imageArea {
   position: relative;
-  flex: 1 1 0;
-  min-height: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.captionWrapper {
+  display: flex;
+  z-index: var(--elevation-base);
+  max-width: 600px;
+  margin: 0 auto var(--space-larger);
+  text-align: center;
+  flex-direction: column;
+  align-items: center;
+  gap: var(--space-small);
+}
+
+.caption {
+  min-height: calc(21.438px * 3);
+  text-align: left;
+  @media (--medium-screens-and-up) {
+    min-height: calc(21.438px * 2);
+  }
 }
 
-.overlay {
+.blurOverlay {
   position: fixed;
   top: 0;
   right: 0;
   bottom: 0;
   left: 0;
-  background-color: var(--color-overlay);
-}
-
-.image {
-  position: absolute;
-  top: 50%;
-  max-width: 100%;
-  max-height: 100%;
-  margin: auto;
+  width: 100%;
+  height: 100%;
```

Comment:
I think these height/width declarations are redundant no? since we're already telling it to be fixed + top,right,bottom, and left of `0` - effectively telling it to completely fill the space